src/app/users/users-update/users-update.component.ts

```diff
@@ -44,6 +44,8 @@ export class UsersUpdateComponent implements OnInit {
   file: any;
   roles: string[] = [];
   languages = languages;
+  minDate = new Date(2000, 0, 1);
```

Comment:
you have min date set 2000 and max date is -30 which is less that 2000 so this does not work at all.
We do not need min date also max date should need to be today.
We do not want to have more control on DOB.